File: DeJia/app/src/main/java/com/dejia/anju/api/BindJPushApi.java
package com.dejia.anju.api;

import android.content.Context;

import com.dejia.anju.api.base.BaseCallBackApi;
import com.dejia.anju.api.base.BaseCallBackListener;
import com.dejia.anju.net.NetWork;

import java.util.Map;

//绑定极光账号id
public class BindJPushApi implements BaseCallBackApi {

    @Override
    public void getCallBack(Context context, Map<String, Object> maps, final BaseCallBackListener listener) {
        NetWork.getInstance().call("message", "jPushBind", maps, mData -> listener.onSuccess(mData));
    }
}

File: DeJia/app/src/main/java/com/dejia/anju/api/BindJPushApi.kt
package com.dejia.anju.api

import android.content.Context
import com.dejia.anju.api.base.BaseCallBackApi
import com.dejia.anju.api.base.BaseCallBackListener
import com.dejia.anju.net.NetWork
import com.dejia.anju.net.ServerData

//绑定极光账号id
class BindJPushApi : BaseCallBackApi {
    override fun getCallBack(context: Context, maps: MutableMap<String, Any>, listener: BaseCallBackListener<*>) {
        NetWork.instance?.call("message", "jPushBind", maps, context) { mData: ServerData? -> listener.onSuccess(mData) }
    }
}